### DISCUSSION TITLE
From embedchain import App Not working | Basic setup not working

### DISCUSSION THREAD

**Discussion by avi-arora** [mem0ai/mem0#1211] [Q&A] [answered]
Hi Everyone, 

I've tried installing embedchain via pip
pip install embedchain

When i use the basic setup it is not working. 
i am using python 3 latest stable version.

Please refer the below screenshot. 

![image](https://github.com/embedchain/embedchain/assets/10464065/3a942865-0552-4c92-9967-86c044d60b94)

Please help.

**Comment by deven298** [comment]
Hey @avi-arora, welcome to embedchain.

Can you please share your python and embedchain version by running the following commands?

`pip freeze | grep embedchain`
`python -V`

**Reply by avi-arora** [reply]
@deven298 
below is the requested details. 

i am using windows machine, is that a problem?

![image](https://github.com/embedchain/embedchain/assets/10464065/155530bc-5325-47c8-bd72-646d3900244c)
![image](https://github.com/embedchain/embedchain/assets/10464065/fbf45823-cdb6-4f69-b44d-ff1e9c6c5eb9)

**Reply by deven298** [reply, marked as answer]
@avi-arora there are two problems here.

1. We are not able to support python 3.12 as other dependencies have not yet caught up with the latest python version. Please downgrade python version to 3.11
2. Also your embedchain version is 0.0.1 which is the first version of embedchain. Please upgrade your embedchain version to the latest version 0.1.68 with `pip install -U embedchain` (do this after after downgrading your python version.)
